### ISSUE TITLE
[AMD] Bug in Graphic - Issue "Devil's Hand" in Shenmue 2 

### ISSUE THREAD

**Issue by TommyThuyen** [flyinghead/flycast#879]
**Please Note: This form is the minimum required information for submitting bugs.**  
**Removing this form may lead to your issue being closed until it is completed.**

Platform / OS / Hardware: 
Windows 11
AMD Ryzen7 6800U RAM 32GB
GPD WinMax 2

Github hash: 1e195b72
Flycast ver: 2.0-241-g1e195b72

**_Description of the Issue_**

Hands of characters disappears or turned into "Devil's Hand" in some scene.

**_Debugging Steps Tested_**

**_Logs Gathered_**

```

Please paste the contents of the log / logs here
You may leave this field blank if you have none.

```

**_Screenshots & SaveFile_**

![Flycast_Settings](https://user-images.githubusercontent.com/122481706/212562177-7316aaa4-8ec0-4874-8793-f09efc51f028.png)
![Bug_DevilHand4](https://user-images.githubusercontent.com/122481706/212562194-d011837e-db7f-45f6-947b-cbd84c39462d.png)
![Bug_DevilHand5](https://user-images.githubusercontent.com/122481706/212562208-662fc49e-37f3-4eca-a3bd-23a89768acdd.png)
![Bug_DevilHand1](https://user-images.githubusercontent.com/122481706/212562232-0e05eab8-53ea-42b8-b67b-c075cf7d9305.png)
[Save File](https://www.mediafire.com/file/njmmhhpd769ncf6/shenmue2_save.zip/file)

**Comment by flyinghead** [comment]
Can you upload a savestate showing one on these scenes?

**Comment by TommyThuyen** [comment]
> Can you upload a savestate showing one on these scenes?

thank you for quick response. Could you please guide me what's it default savestate folder in MS Windows ?

**Comment by flyinghead** [comment]
Save states are created in the `data` subfolder in the Flycast home folder. They have a `.state` extension.

**Comment by TommyThuyen** [comment]
> Save states are created in the `data` subfolder in the Flycast home folder. They have a `.state` extension.

Thank you. I've added a Save file url in the start topic above.

**Comment by flyinghead** [comment]
Thank you for the savestates.
Unfortunately I don't see anything wrong with the hands on my machine/GPU, even when using the same settings as you.

This could specific to AMD GPUs since I'm testing with NVidia.

Does the issue happen also with OpenGL and DirectX11?
Also try disabling `Native Depth Interpolation`.

**Comment by TommyThuyen** [comment]
[](url)> Thank you for the savestates. Unfortunately I don't see anything wrong with the hands on my machine/GPU, even when using the same settings as you.
> 
> This could specific to AMD GPUs since I'm testing with NVidia.
> 
> Does the issue happen also with OpenGL and DirectX11? Also try disabling `Native Depth Interpolation`.

GPU AMD Radeon 680M (CPU AMD 6800U). This issue is happened whether it's running on DX11 or OpenGL or Vulkan and already disabled `Native Depth Interpolation`.

![image](https://user-images.githubusercontent.com/122481706/215286558-335fbdb6-7501-4ed9-865b-9a58f3680b57.png)

**Comment by Nejsplitter** [comment]
i also get this issue on xbox series x, tried with per pixel, enabling/disabling native depth interpolation, issue persists

**Comment by Vinny460** [comment]
I am having this rendering issue on the latest version of flycast on either my pc or laptop. My laptop has an rtx 3060 gpu and a ryzen 5 5600h cpu and my desktop has a gtx 1070 and a i7 6700k so what happened was i downloaded the updated version of flycast on my desktop and noticed that ryo's hands were shown exactly like in the pictures above so i also tried retroarch flycast core all updated core and retroarch build exact same thing hands render the same so what i did was check on my laptop and i have retroarch already installed there with flycast setup and ryo's hands render normally so what i did was copy over that exact install and build to a usb and copied it over to my desktop pc and i no longer have that devil hand issue on my pc. also i just installed flycast on my nintendo switch with the latest version and i am now having that devilhand rendering problem so it appears to me that the latest flycast version has an issue with shenmue 2 i don't know

**Comment by Vinny460** [comment]
lol issue fixed i installed flycast 2.0 with naomi support and bang no more devil hand
i knew it was the updated version that broke shenmue 2

**Comment by Nejsplitter** [comment]
just updated to v2.1 265 g4ca85e959 and issue is gone on xbox series x
